### PR TITLE
Added new section to increase visibility of feature for cloud services

### LINKED ISSUE #4988: Add information to enable the AWS module
### Documentation
Right now, within [Monitoring AWS based services](https://documentation-dev.wazuh.com/4.3-rc/amazon/services/index.html) section of the documentation, there is no explanation on how to enable the Wazuh AWS module in the UI. In fact, its existence is not even clearly mentioned.

This could mean that many users do not activate it or do not know how to do it. I think it would be useful to add a new point in [Considerations for configuration](https://documentation-dev.wazuh.com/4.3-rc/amazon/services/prerequisites/considerations.html) to explain the steps for its activation and some image of the dashboard:

image

### Blogpost
In step 4 of this blogpost it is explained how to activate the AWS dashboard but the way to do it is now outdated. It should be changed too:
- [Integrating AWS CloudTrail in Wazuh.](https://wazuh.com/blog/integrating-aws-cloudtrail-in-wazuh/)

### PR DESCRIPTION
**Pull request by EduLeon12**
|Related issue|
|---|
|Closes #4988 |

## Description

As described in #4988, there was no documentation regarding the feature to enable the _Security Information_  dashboard not only in AWS but in all Cloud modules sections that support this feature.

Documentation has been updated to enhance the visibility of this feature to the user.

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/amazon/services/prerequisites/considerations.rst [hunk 1]
```diff
@@ -222,3 +222,7 @@ The following is an example of a valid configuration.
     </service>
 
   </wodle>
+  
+.. note::
+   Once the configuration is complete, you can enable the `Security Information` visualization feature in the **Wazuh Dashboard**. You can read more about it `here <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_.
+
```
Comment (nico-stefani) [inline source/amazon/services/prerequisites/considerations.rst:228]: The note has little visibility inside of **FIPS endpoints**. I suggest adding a new section with this content.

File: source/gcp/prerequisites/considerations.rst [hunk 2]
```diff
@@ -83,3 +83,7 @@ Below there is an example of a configuration that uses more than one bucket:
     </bucket>
 
   </gcp-bucket>
+  
+.. note::
+   Once the configuration is complete, you can enable the `Security Information` visualization feature in the **Wazuh Dashboard**. You can read more about it `here <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_.
+
```
Comment (nico-stefani) [inline source/gcp/prerequisites/considerations.rst:89]: Like in AWS, the note here has a lack of visibility. I suggest adding a new section with this content.

**Review by nico-stefani — CHANGES_REQUESTED** [review]

**Review by nico-stefani — APPROVED** [review]
LGTM

File: source/amazon/services/prerequisites/considerations.rst [hunk 3]
```diff
@@ -222,3 +222,10 @@ The following is an example of a valid configuration.
     </service>
 
   </wodle>
+  
+  
+Enabling dashboard visualization  
+--------------------------------
+  
+   Once the configuration is complete, you can enable the `Security Information` visualization feature in the **Wazuh Dashboard**. You can read more about it `here <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_.
```
Comment (s-ocando) [inline source/amazon/services/prerequisites/considerations.rst:230]:
```suggestion
Once the configuration is complete, you can enable the corresponding Security Information Management module on the Wazuh Dashboard. To learn more, see the`modules <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_ section.
```

Comment (s-ocando) [reply source/amazon/services/prerequisites/considerations.rst:230]: Consider adding the corresponding screenshot to make it more appealing to users. For example:

image

**Review by s-ocando — COMMENTED** [review]
Hi @EduLeon12, 

Please review my suggestions and if you agree, apply them in all the corresponding cases.

**Comment by EduLeon12** [comment]
Hi @s-ocando.

All changes have been addressed and dashboard screenshot added to every service.

Will wait for your review again.

Thank You.

**Review by s-ocando — APPROVED** [review]

**Review by davidjiglesias — APPROVED** [review]

**Pull request merged**